addons/tmf_loadouts/loadouts/UK_Army_2005.hpp

```diff
@@ -0,0 +1,544 @@
+		LIST_2("CUP_HandGrenade_L109A1_HE"),
+		LIST_2("SmokeShell")
+	};
+};
+
+class g : r {
+	displayName = "Grenadier";
+	vest[] = {"CUP_V_B_BAF_DDPM_Osprey_Mk3_Grenadier"};
+	primaryWeapon[] = {"CUP_arifle_L85A2_GL"};
+	scope[] = {};
+	magazines[] = {
+		LIST_6("CUP_30Rnd_556x45_Stanag"),
+		LIST_2("CUP_30Rnd_556x45_Stanag_Tracer_Red"),
+		LIST_2("CUP_HandGrenade_L109A1_HE"),
+		LIST_2("SmokeShell"),
+		LIST_10("CUP_1Rnd_HEDP_M203"),
+		LIST_2("CUP_1Rnd_Smoke_M203")
+	};
+};
+
+class car : r {
+	displayName = "Carabinier";
+};
+
+class cls : car {
+	displayName = "Combat Life Saver";
+	traits[] = {"medic"};
+	vest[] = {"CUP_V_B_BAF_DDPM_Osprey_Mk3_Medic"};
+	backpack[] = {"CUP_B_Bergen_BAF"};
+	backpackItems[] = { MEDICAL_CLS };
+};
+
+class m : cls {
+	displayName = "Medic";
+	backpack[] = {"CFP_Carryall_DPMWoodland"};
+	backpackItems[] = { MEDICAL_M };
+};
+
+class smg : r {
+	displayName = "Submachinegunner";
+	primaryWeapon[] = {"CUP_smg_MP5A5"};
+	scope[] = {};
+	magazines[] = {
+		LIST_5("CUP_30Rnd_9x19_MP5"),
+		"CUP_HandGrenade_L109A1_HE",
+		"SmokeShell"
+	};
+};
+
+class ftl : g {
+	displayName = "Fireteam Leader";
+	items[] += {"ACE_MapTools"};
+	magazines[] += {
+		LIST_2("1Rnd_SmokeRed_Grenade_shell"),
+		LIST_2("CUP_1Rnd_StarFlare_Red_M203"),
+		"SmokeShellGreen"
+	};
+	linkedItems[] += {
+		"Binocular"
+	};
+};
+
+class sl : ftl {
+	displayName = "Squad Leader";
+	vest[] = {"CUP_V_B_BAF_DDPM_Osprey_Mk3_Officer"};
+	backPack[] = {"CUP_B_SLA_Medicbag"};
+	sidearmWeapon[] = {"hlc_pistol_P226US"};
+	magazines[] += {
+		LIST_3("hlc_15Rnd_9x19_B_P226")
+	};
+	linkedItems[] = {
+		"ItemMap",
+		"ItemCompass",
+		"ItemRadio",
+		"ItemWatch",
+		"ACE_Vector",
+		"ItemGPS"
+	};
+};
+
+class co : sl {
+	displayName = "Platoon Leader";
+	headgear[] = {"CUP_H_BAF_PARA_PRROVER_BERET"};
+};
+
+class fac : co {
+	displayName = "Forward Air Controller";
+	vest[] = {"CUP_V_B_BAF_DDPM_Osprey_Mk3_Scout"};
+	backpack[] = {"CUP_B_Predator_Radio_MTP"};
+	backpackItems[] = {
+		FAC_GEAR,
+		LIST_2("CUP_1Rnd_SmokeRed_M203"),
+		LIST_4("CUP_1Rnd_SmokeGreen_M203")
+	};
+	linkedItems[] = {
+		"ItemMap",
+		"ItemCompass",
+		"ItemRadio",
+		"ItemWatch",
+		"CUP_SOFLAM",
+		"ItemGPS"
+	};
+};
+
+class ar : r {
+	displayName = "Automatic Rifleman";
+	vest[] = {"CUP_V_B_BAF_DDPM_Osprey_Mk3_AutomaticRifleman"};
+	backPack[] = {"CUP_B_AlicePack_Khaki"};
+	primaryWeapon[] = {"CUP_lmg_minimipara"};
+	sidearmWeapon[] = {"hlc_pistol_P226US"};
+	magazines[] = {
+		LIST_4("CUP_200Rnd_TE4_Red_Tracer_556x45_L110A1"),
+		"CUP_HandGrenade_L109A1_HE",
+		"SmokeShell",
+		LIST_3("hlc_15Rnd_9x19_B_P226")
+	};
+};
+
+class aar : r {
+	displayName = "Assistant Automatic Rifleman";
+	backPack[] = {"usm_pack_200rnd_556_bandoliers"};
+	backpackItems[] = {
+		LIST_4("CUP_200Rnd_TE4_Red_Tracer_556x45_L110A1"),
+	};
+	linkedItems[] += {"Binocular"};
+};
+
+class rat : car {
+	displayName = "Rifleman (AT)";
+	secondaryWeapon[] = {"CUP_launch_NLAW"};
```

Comment:
NLAW came into British service in 2009 https://en.wikipedia.org/wiki/MBT_LAW 

AT4 (ILAW) would likely be used instead